### DISCUSSION TITLE
H265 record playback is not supported in the browser?

### DISCUSSION THREAD

**Discussion by t0ny-peng** [blakeblackshear/frigate#3638] [General]
Hey guys. I have a Dahua HDW4631C-A camera. The main stream is set to use H265 for recording and the sub stream is set to H264 for detection.

<img width="1217" alt="Screen Shot 2022-08-13 at 10 21 37 PM" src="https://user-images.githubusercontent.com/4816327/184523706-b6d97629-622a-4ae4-9f54-dbb9ced9c4ec.png">

Object detection works fine with the substream. However, when an object is detected, I'm not able to preview the clip in the browser, with this error message.
<img width="1454" alt="image" src="https://user-images.githubusercontent.com/4816327/184523754-cb1d72f2-e0be-4b6c-8aaf-f21b29f7e04f.png">

If I click the download button, the video can be played in VLC, which confirms that the format is H265 HEVC. If I change the main stream to H264, there's no problem previewing it in the browser. Is this a known limit? I can live with it for now since I uploaded all the videos to the cloud, but it'd be nice to have a quick glance if necessary.

Btw, I'm using the beta version of 0.11.0 in a Docker container with Google Coral USB.
Thanks!

**Comment by trythatagain** [comment]
I believe its a browser library problem. I think Edge can view h265 and maybe chrome. There are a number of threads on here discussing it  and how to recode to h264.

**Reply by NickM-27** [reply]
It's a licensing issue, h.265 is very expensive. Edge & Safari at one point could playback h.265 by sending the entire pipeline to a gpu (which has the license) but chrome, FF, Opera, etc. can not play h.265. Nothing frigate can do and that's likely to never change especially as AV1 is coming and is open source.

**Comment by banthungprong** [comment]
I just installed a chromium beta v109. Since 108 there should be Linux/Ubuntu-Support for H.265.
And its working!
Now I can play my recordings without problems in the Frigate. 
Even H.265+ Streams from my HikVision NVR and Cams!
![Dome](https://user-images.githubusercontent.com/59219161/196691808-fbc4e55f-3303-4eb7-bb38-c9e47e66abeb.jpg)
Full-HD Recordings are ok, this camera will record from now on with 4K
